<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>758</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DAMAKER</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/files.qrc">
    <normaloff>:/16x16/icons/16x16/logo_dmkr.png</normaloff>:/16x16/icons/16x16/logo_dmkr.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QSplitter" name="splitter_horizontal">
   <property name="styleSheet">
    <string notr="true">QFrame {
	border: 0px;
}

/* ----  TAB WIDGET ---- */

QTabWidget {
	background-color: rgb(218, 219, 221);
	padding: 15px;
	margin: 10px;
}

QTabBar {
	background-color: rgb(0, 0, 0);
	padding: 15px;
	margin: 10px;
font-weight: bold;
}

QTabBar::pane {
  border: none;
}

QTabBar::tab {
    color: black;  
	background-color: rgb(220, 220, 220);
    border: 1px solid rgb(235, 235, 235); 
    padding: 4px 10px 4px 10px;
   border-radius: 2px;
} 

QTabBar::tab:hover {
  background-color: rgb(227, 241, 240);
} 

QTabBar::tab:selected { 
	background-color: rgb(222, 237, 255);
}

/* ---- SLIDER ---- */

QSlider::groove:horizontal {
    border-radius: 4px;
    height: 18px;
	margin: 0px;
	background-color: rgb(72, 75, 75);
}

QSlider::handle:horizontal {
    background-color: rgb(35, 31, 32);
    border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 2px;
}
QSlider::handle:horizontal:hover {
    background-color: rgb(42, 42, 42);
}

QSlider::groove:vertical {
    border-radius: 9px;
    width: 18px;
    margin: 0px;
	background-color: rgb(52, 59, 72);
}

QSlider::handle:vertical {
    background-color: rgb(35, 31, 32);
	border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 2px;
}
QSlider::handle:vertical:hover {
    background-color: rgb(42, 42, 42);
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QSplitter" name="splitter_vertical1">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="ContentFrame" name="dock1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="layout_dock1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="dock1_tab">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <widget class="QFrame" name="dock1_tabPage1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(244, 248, 249);
margin: 0px;</string>
         </property>
         <attribute name="title">
          <string>Page</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QSlider" name="horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListView" name="listView"/>
          </item>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>738</width>
               <height>307</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="ContentFrame" name="dock3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="layout_dock3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="dock3_tab"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_vertical2">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="ContentFrame" name="dock2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="layout_dock2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="dock2_tab"/>
      </item>
     </layout>
    </widget>
    <widget class="ContentFrame" name="dock4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="layout_dock4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="dock4_tab"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContentFrame</class>
   <extends>QFrame</extends>
   <header>damaker_gui.widgets.ContentFrame</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/files.qrc"/>
 </resources>
 <connections/>
</ui>
